client/src/components/settings/honesty.test.tsx

```diff
@@ -1,40 +1,27 @@
-import { Button } from '@freecodecamp/react-bootstrap';
```

Comment:
changed this test to use `@testing-library/react` which resolves all linting errors